de.metas.business.rest-api-impl/src/main/java/de/metas/rest_api/invoicecandidates/impl/InvoiceJsonConverterService.java

```diff
@@ -41,5 +46,16 @@ public final JsonInvoiceCandCreateResponse toJson(@NonNull final IInvoiceCandida
 				.workpackageQueueSizeBeforeEnqueueing(enqueueResult.getWorkpackageQueueSizeBeforeEnqueueing()).build();
 		return JsonInvoiceCandCreateResponse.ok(jsonInvoiceCand);
 	}
+	
+	public List<ExternalHeaderAndLineId> convertJICToExternalHeaderAndLineIds(
```

Comment:
i just took a note to add `final` `@NonNull` here some time this week, but it's not an R&I-stopper

Reply:
done